dmf/config.py

```diff
@@ -4,3 +4,5 @@
 PASSWORD=None
 HTTPS_PROXY=None
 HTTP_PROXY=None
+SSN_CLIENT=None
+SSN_MODULEN=None
```

Comment:
This is a typo -- should be `SSN_MODULE` (without the `N`).  I'll fix.

Reply:
Done in commit b0d7baefd3f.  (It will all get washed away in the move to `dmf/config.py.tmpl`, which is coming in moment via merge, but I wanted the discrepancy between the old file and the new file to be resolved before that.)